frame/benchmarking/src/lib.rs

```diff
@@ -160,6 +160,12 @@ pub use v1::*;
 /// The underscore will be substituted with the name of the benchmark  (i.e. the name of the
 /// function in the benchmark function definition).
 ///
+/// In case of a `force_origin`, this is the general syntax:
```

Comment:
perhaps a sentence or two explaining what `force_origin` means or a link to elsewhere in the docs where this is explained

Reply:
Updated. Please let me know if this works.